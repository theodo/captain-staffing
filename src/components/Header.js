import React from 'react'

import trello from '../assets/trello.jpg'

export default class Header extends React.Component {

  render() {
    return (
      <div>
        <a
          className="trelloLink"
          href="https://trello.com/b/4p4llsMz/captain-staffing"
          target="_blank"
          rel="noopener noreferrer"
        >
          <img
            className="trelloLink__img"
            alt="trello"
            src={trello}
          />
        </a>
        <h1 className="brand">
          Captain Staffing
        </h1>
      </div>
    )
  }
}
