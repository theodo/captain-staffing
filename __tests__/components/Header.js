import Header from 'components/Header'
import React from 'react'

import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

describe('Header', () => {
  it('should render a title and a trello link', () => {
    const header = shallow(<Header />)

    expect(toJSON(header)).toMatchSnapshot()
  })
})
