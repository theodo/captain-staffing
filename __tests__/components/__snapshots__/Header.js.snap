exports[`Header should render a title and a trello link 1`] = `
<div>
  <a
    className="trelloLink"
    href="https://trello.com/b/4p4llsMz/captain-staffing"
    rel="noopener noreferrer"
    target="_blank">
    <img
      alt="trello"
      className="trelloLink__img"
      src="fileMock" />
  </a>
  <h1
    className="brand">
    Captain Staffing
  </h1>
</div>
`;
